### PR TITLE
Initial card and card description styling

### PR DESCRIPTION
**Pull request by Cody-Price**
Hey guys, got some initial stying on the cards done with the wiggle on hover animation, in case you guys would like how they may roughly look as you are styling

### CODE REVIEW

File: src/Card-Container.css [hunk 1]
```diff
@@ -0,0 +1,8 @@
+.card-container {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, 225px);
+  grid-gap: 20px;
+  justify-content: space-around;
+  padding: 25px;
```
Comment (kayleenovak) [inline src/Card-Container.css]: The padding and padding left is repetitive, can the padding-left be combined into the padding?

File: src/Card.css [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+.card:hover > .dog-img {
+  opacity: 0.6;
+  filter: blur(1px);
+}
+
+.card:hover > .card-desc {
+  text-decoration: blur;
+}
+
+.card:hover {
```
Comment (kayleenovak) [inline src/Card.css]: The animation looks great!

File: src/Card.css [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+@-webkit-keyframes wiggle {
+  0% {-webkit-transform: rotate(2deg);}
+  25% {-webkit-transform: rotate(-2deg);}
+  50% {-webkit-transform: rotate(5deg);}
+  75% {-webkit-transform: rotate(-2deg);}
+  100% {-webkit-transform: rotate(0deg);}
+}
+
+/*@-ms-keyframes wiggle {
```
Comment (kayleenovak) [inline src/Card.css]: Are these keyframes that are being used? If not, can the commented out chunks get removed?

File: src/Card-Container.css (hunk 1 above)
Comment (Cody-Price) [reply src/Card-Container.css]: Thank you Kaylee! got it fixed

File: src/Card.css (hunk 2 above)
Comment (Cody-Price) [reply src/Card.css]: Appreciate the compliment :)

File: src/Card.css (hunk 3 above)
Comment (Cody-Price) [reply src/Card.css]: you are definitely right, ill be a bit more thorough on my review later in the night or maybe just push them in the morning when im more awake, but i got those taken out for you :)

**Comment by Cody-Price** [comment]
Hey Kaylee, up for resubmission :)

**Comment by kayleenovak** [comment]
Great, thank you for updating that!

**Pull request merged**